app/services/second_renewal_reminder_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SecondRenewalReminderService < RenewalReminderServiceBase
+
+  private
+
+  def send_email(registration)
+    RenewalReminderMailer.second_reminder_email(registration).deliver_now
```

Comment:
The first email reminder is controlled from a feature toggle. The second should be as well (at least initially) particularly as we have told NCCC that it can be switched off with a simple environment config change.

I think in the next release we can start to look at removing the feature toggles of things that no longer require them.

Reply:
Added in the rake task as per first email reminder.